syntax = "proto3";

service Grpclb {
    rpc Execute (TxRequest) returns (TxResponse) {
    }
    rpc Query (TxRequest) returns (TxResponse) {
    }
}

message TxRequest {
    string channelName = 1;
    string chaincodeName = 2;
    string functionName = 3;
    repeated string args = 4;
    int32 numOfTxs = 5;
}

message TxResponse {
    string response = 1;
}
